presentation/reveal.md

```diff
@@ -265,7 +265,7 @@ Note: // Todo, add tip for opening console fast. We're going to do it alot.
 - Use `Observable.range()` to write your first Observable, and log its contents to your console.
 
 ```js
-var range = Observable.range(3); // Counts to `x`
+var range = Observable.range(1, 3); // Counts to `x`
```

Comment:
Werkt `range(3)` niet meer? En hier gebruiken we ook nog de `Observable.range()`, is dat in lijn met de v6 wijzigingen?